modin/pandas/test/dataframe/test_binary.py

```diff
@@ -163,9 +163,16 @@ def test_math_alias(math_op, alias):
 @pytest.mark.parametrize("op", ["eq", "ge", "gt", "le", "lt", "ne"])
 @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
 def test_comparison(data, op, other):
+    def operation(df):
+        df = getattr(df, op)(df if other == "as_left" else other)
+        if other == "as_left" and StorageFormat.get() == "Hdk":
+            # In case of comparison with a DataFrame, HDK returns
+            # a DataFrame with sorted columns.
+            df = df.sort_index(axis=1)
```

Comment:
Sorry, forgot `return`.
```suggestion
            df = df.sort_index(axis=1)
        return df
```

Reply:
Didn't notice that either. Thanks, I've already rebased the previous commit.